### ISSUE TITLE
numbers in header name

### ISSUE THREAD

**Issue by edclement** [joostfarla/serverless-cors-plugin#4]
The current validation for header names (line ~100) is:
```
allowHeaders: Joi.array().min(1).items(Joi.string().regex(/^[a-zA-Z-]+$/))
```
We have a case where our header contains numbers. Amazon requires a special header when using signed (authenticated) requests: `X-Amz-Content-Sha256`. http://docs.aws.amazon.com/general/latest/gr/signature-version-4.html. The current validation regex fails on the #'s in the header name.

I think a more appropriate regex would be `/^(\w|-)+$/`.

**Comment by joostfarla** [comment]
Fixed in release v0.1.2!

**Comment by edclement** [comment]
Thanks!
